examples/generalized-compute/split-to-multiple-parts/split_dataset.py

```diff
@@ -0,0 +1,43 @@
+import os
+import json
+import pandas as pd
+import numpy as np
+
+DEFAULT_NUM_PARTS = 3
+
+if __name__ == '__main__':
+    print("Starting")
+
+    # Read the number of expected parts from the input parameters
+    try:
+        with open('/input/run_params.json') as params_file:
+            params_json = json.load(params_file)
+    except FileNotFoundError:
+        num_parts = DEFAULT_NUM_PARTS
+    else:
+        num_parts = params_json.get('num_parts', DEFAULT_NUM_PARTS)
+    print(f"Got {num_parts=}")
+
+    try:
+        num_parts = int(num_parts)
+    except ValueError:
+        print(f"Invalid value for num_parts (must be a positive integer): {num_parts}")
+        exit(1)
+
+    if num_parts < 1:
+        print(f"Invalid value for num_parts (must be a positive integer): {num_parts}")
+        exit(1)
+
+    # Read the input data from /input and split it using np.array_split
+    df = pd.read_csv('/input/dataset.csv')
```

Comment:
Isn't it better to use `/input/0/dataset.csv`?

Reply:
Both can work - updated to `/input/0/dataset.csv`